### PR TITLE
Post fix for hosted happs

### PR DESCRIPTION
**Pull request by zo-el**
a tmp holochain optimization: holochain/holochain#1059

- Update the post-fix to hosted happ id with the string `###zero###`

- [ ] I have updated `CHANGELOG.md` to include all notable changes.
- [] I have checked that `README.md` only contains up-to-date information.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Changelog
+
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [#153](https://github.com/Holo-Host/holo-envoy/pull/153)
+
+### Changed
+
+- a tmp holochain optimization: [holochain/holochain#1059](https://github.com/holochain/holochain/pull/1059)
+  - Update the post-fix to hosted happ id with the string `###zero###`
```
Comment (timotree3) [inline CHANGELOG.md:10]: Can we specify anonymous in the changelog?
```suggestion
  - Update the installed app ids of anonymous instances to end with the string `###zero###`
```

**Review by timotree3 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zo-el) [reply CHANGELOG.md:10]: This is for the `Hosted agent happ instance` not the anonymous instance

Comment (timotree3) [reply CHANGELOG.md:10]: oh you're right. carry on :)

**Comment by timotree3** [comment]
wait I shouldn't have approved this. CI is failing

**Pull request merged**